dct_backend/webscraper.py

```diff
@@ -0,0 +1,119 @@
+import requests
+from bs4 import BeautifulSoup
+import dct
+
+# DCT import
```

Comment:
Please remove the commented code -- if you need them as notes, copy the file and paste it outside of the project directory. They make reading the code confusing as to what is relevant and what isn't